ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -188,7 +198,11 @@ open class PillButton: UIButton {
                     setTitleColor(style.selectedDisabledTitleColor(for: window), for: .normal)
                 }
             } else {
-                backgroundColor = style.backgroundColor(for: window)
+                if useCustomBackgroundColor {
```

Comment:
nit: if we're only setting the backgroundColor here, I think a ternary operator makes more sense....

Reply:
Done

---
In reply to: [468244410](https://github.com/microsoft/fluentui-apple/pull/175#discussion_r468244410) [](ancestors = 468244410)